Snippets/StructureMap/Extensions.DependencyInjection_1/Extensions.DependencyInjection_1.csproj

```
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net462</TargetFramework>
+    <LangVersion>7.1</LangVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="NServiceBus.Extensions.DependencyInjection" Version="1.*" />
+    <PackageReference Include="StructureMap.Microsoft.DependencyInjection" Version="2.0.0" />
```

```suggestion
    <PackageReference Include="StructureMap.Microsoft.DependencyInjection" Version="2.*" />
```

should this use a wildcard dependency too?